src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -149,6 +164,27 @@ export default {
                     || (scrollHeight - scrollTop) > 2 * clientHeight;
             this.autoScroll = !disableScroll;
         },
+        loadVisibleLayers(metaDataValues) {
+            let layersMetadata = metaDataValues.layers;
+            if (layersMetadata) {
+                this.layers = layersMetadata;
+                if (this.domainObject.configuration) {
+                    let persistedLayers = this.domainObject.configuration.layers;
+                    layersMetadata.forEach((layer) => {
+                        const persistedLayer = persistedLayers.find((object) =>{ return object.name === layer.name; });
```

Comment:
This could probable be split over two lines.

Reply:
OK